Goal: Fix the su synopsis. username is referenced in the manpage, not
      LOGIN

Fixes: 501830

Status wrt upstream: Will be fixed in 4.1.3

--- a/man/su.1.xml
+++ b/man/su.1.xml
@@ -38,7 +38,7 @@
       </arg>
       <arg choice='opt'>
 	<arg choice='plain'>
-	  <replaceable>LOGIN</replaceable>
+	  <replaceable>username</replaceable>
 	</arg>
       </arg>
     </cmdsynopsis>
